css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
```

Comment:
Aaaah, alright! Then, how about this: you're just interested in having the image be `200px` wide, correct? So, try to leave the `width` rule in, but remove the `height` altogether. Does the image still maintain its original aspect ratio? :)